aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/codegen/events/RefreshSessionEvent.kt

```diff
@@ -25,6 +26,11 @@ class RefreshSessionEvent(val eventType: EventType, override val time: Date? = n
         data class RefreshUserPoolTokens(val signedInData: SignedInData) : EventType()
         data class RefreshAuthSession(val signedInData: SignedInData, val logins: LoginsMapProvider) : EventType()
         data class RefreshUnAuthSession(val logins: LoginsMapProvider) : EventType()
+        data class RefreshFederatedSession(
+            val federatedToken: FederatedToken,
```

Comment:
Do we really need federated token? It's only used to create logins map which is already passed in the event.

Reply:
This class was removed